includes/utils.php

```diff
@@ -128,10 +130,21 @@ public static function replace_urls( $from, $to ) {
 			"WHERE `meta_key` = '_elementor_data' AND `meta_value` LIKE '[%' ;" ); // meta_value LIKE '[%' are json formatted
 		// @codingStandardsIgnoreEnd
 
+		$second_rows_affected = $wpdb->query(
+			"UPDATE {$wpdb->postmeta} " .
+			$wpdb->prepare( 'SET `meta_value` = REPLACE(`meta_value`, %s, %s) ', $from, $to ) .
+			'WHERE `meta_key` = \'' . Manager::META_KEY . '\''
+		);
+
+		$rows_affected = false === $second_rows_affected ? false : $rows_affected + $second_rows_affected;
```

Comment:
The second query shouldn't break the action.
```suggestion
		if ( $second_rows_affected ) {
			$rows_affected += $second_rows_affected;
		}
```

Reply:
Handled.